gramps/gen/utils/symbols.py

```diff
@@ -0,0 +1,409 @@
+                                      "&#9898;",  ""),
+               (_("Neuter"),                                     '\u26b2',
+                                      "&#9906;",  ""),
+               (_("Illegitim"),                                  '\u229b',
+                                      "&#8859;",  ""),
+               (_("Birth"),                                      '\u002a',
+                                      "&#0042;",  "*"),
+               (_("Baptism/Christening"),                        '\u007e',
+                                      "&#0126;",  "~"),
+               (_("Engaged"),                                    '\u26ac',
+                                      "&#9900;",  "o"),
+               (_("Marriage"),                                   '\u26ad',
+                                      "&#9901;",  "oo"),
+               (_("Divorce"),                                    '\u26ae',
+                                      "&#9902;",  "o|o"),
+               (_("Unmarried partnership"),                      '\u26af',
+                                      "&#9903;", "o-o"),
+               (_("Buried"),                                     '\u26b0',
+                                      "&#9904;", "d"),
+               (_("Cremated/Funeral urn"),                       '\u26b1',
+                                      "&#9905;", "d"),
+               (_("Killed in action"),                           '\u2694',
+                                      "&#9908;", "d"),
+               (_("Extinct"),                                    '\u2021',
+                                      "&#8225;", ""),
+              ]
+
+    # genealogical death symbols
+    DEATH_SYMBOL_NONE                      = 0
+    DEATH_SYMBOL_X                         = 1
+    DEATH_SYMBOL_SKULL                     = 2
+    DEATH_SYMBOL_ANKH                      = 3
+    DEATH_SYMBOL_ORTHODOX_CROSS            = 4
+    DEATH_SYMBOL_CHI_RHO                   = 5
+    DEATH_SYMBOL_LORRAINE_CROSS            = 6
+    DEATH_SYMBOL_JERUSALEM_CROSS           = 7
+    DEATH_SYMBOL_STAR_CRESCENT             = 8
+    DEATH_SYMBOL_WEST_SYRIAC_CROSS         = 9
+    DEATH_SYMBOL_EAST_SYRIAC_CROSS         = 10
+    DEATH_SYMBOL_HEAVY_GREEK_CROSS         = 11
+    DEATH_SYMBOL_LATIN_CROSS               = 12
+    DEATH_SYMBOL_SHADOWED_LATIN_CROSS      = 13
+    DEATH_SYMBOL_MALTESE_CROSS             = 14
+    DEATH_SYMBOL_STAR_OF_DAVID             = 15
+    DEATH_SYMBOL_DEAD                      = 16
+
+    # The following is used in the global preferences in the display tab.
+    #                Name
+    #                              (UNICODE char) (HTML char) SUBSTITUTED string
+    death_symbols = [(_("Nothing"),
+                                    "",         "",        ""),
+                 ("x",
+                                    "x",        "x",       "x"),
+                 (_("Skull and crossbones") +' : \u2620',
```

Comment:
I suggest that you don't add the character to the name.  In English, we wouldn't put a space before the colon.  It would be better to use a translated string format to add the character.

`_('%s: %s)`

Reply:
Done